### PR TITLE
fix: refactoring logger event

### PR DESCRIPTION
**Pull request by elorzafe**
#### Description of changes
Updating event format sent thru `APILoggingProvider`

#### Description of how you validated changes
WIP

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by abhi7cr — APPROVED** [review]
LGTM, thank you 🙂

**Review by abhi7cr — APPROVED** [review]

**Review by cslogan-red — APPROVED** [review]
:+1:

File: packages/core/src/Logger/ConsoleLogger.ts [hunk 1]
```diff
@@ -103,20 +103,30 @@ export class ConsoleLogger implements Logger {
 				obj = obj[0];
 			}
 			strMessage = msg[0];
+
+			if (obj instanceof Error) {
+				error = {
+					message: obj.message,
+					name: obj.name,
+					stackTrace: obj.stack,
```
Comment (manueliglesias) [inline packages/core/src/Logger/ConsoleLogger.ts:111]: Looks like this one is not standarized

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack

**Review by manueliglesias — APPROVED** [review]
LGTM 👍 

We might need to remove the `stackTrace` though

**Pull request merged**